### ISSUE TITLE
Unable to start streamdeck after install

### ISSUE THREAD

**Issue by Noodles4Pasta** [timothycrosley/streamdeck-ui#21]
After following all instructions to install this in POP_OS ( a fork of Ubuntu) and installing all dependencies as needed, the command "streamdeck" does not start the program. All I get is "command not found" in terminal.

I did get a warning that a file needed added to "PATH" and tried to do that, but could still never get "streamdeck" to work. Also there are no instructions or link to instructions on howw to add it to auto start as suggested...

I am stuck and dont know what to try next, as I am a new (ish) user on linux.

(now for the obligatory "dis is broke, fix nao") XD

any help would be greatly appreciated!

**Comment by xorbital** [comment]
The warning is correct. Your shell doesn't know what "streamdeck" is or where to find it, so you'll need to manually add the path where python3 / pip stores it's binaries to your PATH.
Alternatively, you could always specify the absolute path to run streamdeck, this isn't very convenient however. (try `~/.local/bin/streamdeck`, this should always work)
PATH is a variable holding a list of the directories your terminal will look for executables.

To add streamdeck to your PATH, you can modify your terminal config file.  
If you didn't change the default shell, you should be using BASH.  
You can verify this by entering `echo $0` into your terminal - it should say "bash".

Open a terminal and edit your .bashrc file:
```bash
nano ~/.bashc
```
**or** if you like gedit more:
```bash
gedit ~/.bashrc
```
Go to the bottom of the file and add the following:
```
PATH=$PATH:$HOME/.local/bin
```
This will add the /home/user/.local/bin directory to the PATH variable.
Reload the changes:  
Either close and re-open the terminal **or**
```bash
source .bashrc
```
This should resolve your problem.

**Comment by dodgyrabbit** [comment]
Instructions have been merged in #22, closing.
